steering_controllers_library/src/steering_controllers_library.yaml

```
@@ -167,3 +167,10 @@ steering_controllers_library:
     position_feedback is true then ``HW_IF_POSITION`` is taken as interface type",
     read_only: false,
   }
+
+  twist_input: {
+    type: bool,
+    default_value: true,
+    description: "Choose whether a Twist/TwistStamped or a AckermannDriveStamped message is used as input.",
```

```suggestion
    description: "Choose whether a TwistStamped or a AckermannDriveStamped message is used as input.",
```
we don't allow unstamped messages from jazzy on, or am I missing something?

---

Very good point, that was a relic from the original version for Iron. I changed this